mw-webapp/src/component/image/Image.module.scss

```diff
@@ -1,3 +1,15 @@
+.imageWrapperFullScreen {
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  width: 100vw;
+  height: 100vh;
```

Comment:
Isn't is too big image?

Reply:
depending on the orientation of the picture this will change

Reply:
@Ekaterina1994 it is for image wrapper. Image will scale